app/src/main/res/layout/fragment_maps.xml

```diff
@@ -23,4 +23,13 @@
         android:focusable="true"
         android:src="@drawable/baseline_refresh_black_24dp"
         tools:ignore="SpeakableTextPresentCheck" />
+
+    <com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
+        android:id="@+id/fragment_maps_list"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center|top"
+        android:layout_margin="16dp"
+        android:contentDescription="@string/view_nearby_codes"
+        android:text="@string/view_nearby_codes" />
```

Comment:
I added the button here, I'm not sure why it's not showing up.

It might be because it's an Extended FAB (extended adds the text to it), but I doubt it cause it worked on my end.